Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -18,6 +19,24 @@ public abstract class Cell : Element, ICellController
 
 		bool _nextCallToForceUpdateSizeQueued;
 
+		EffectiveFlowDirection _effectiveFlowDirection = EffectiveFlowDirection.LeftToRight | EffectiveFlowDirection.Implicit;
+		EffectiveFlowDirection IFlowDirectionController.EffectiveFlowDirection
```

Comment:
as you're implementing this explicitly, I guess this is not meant to be set directly by the user, so there's no point in making this property bindable ?

Reply:
Correct